WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift

```diff
@@ -123,8 +130,10 @@ public class BlockEditorScreen: ScreenObject {
     }
 
     private func addBlock(_ blockLabel: String) {
+        let blockButton = XCUIApplication().buttons[blockLabel]
         addBlockButton.tap()
-        XCUIApplication().buttons[blockLabel].tap()
+        guard blockButton.waitForIsHittable(timeout: 3) else { return }
```

Comment:
Would we want the test to fail at this point? That is, we're waiting for the button to be hittable to avoid a failure, but if it is not hittable, then the app is not in the expected state and we should fail the test as soon as possible.

```suggestion
        XCTAssertTrue(blockButton.waitForIsHittable(timeout: 3))
```

Or am I missing something?

Reply:
Yeah, you're right.